api/controllers/tasks.py

```diff
@@ -120,6 +125,23 @@ def process_proposal(credentials, tpid):
             last_updated=db.sql.func.now(),
         )  # Annotation config is sentiment example.
 
+        emailReceiver = tp_creator_email
+        cc = "no-reply@dynabench.org"
+        subject = "Dynabench Task Approval Mail"
+
+        try:
+            mail.send(
+                server="",
+                config=config,
```

Comment:
commenting on this in conjunction my comment for line 134 - you can just retrieve the config from `bottle.default_app()`, there's no need to import from the original `config.py` (line 17 `from common.config import config`) file that server.py subsequently loads the bottle config from